include/onnxruntime/core/framework/kernel_registry.h

```diff
@@ -25,7 +25,7 @@ class KernelRegistry {
   // itself.
   // TODO(Task:132) Make usage of unique_ptr/shared_ptr as out param consistent
   Status TryCreateKernel(const onnxruntime::Node& node, const IExecutionProvider& execution_provider,
-                         const std::unordered_map<int, OrtValue>& initialized_tensors,
+                         const std::unordered_map<int, OrtValue>& constant_initialized_tensors,
```

Comment:
>constant_initialized_tensors [](start = 66, length = 28)

Are these changes necessary? IMO we should minimize changes to framework if possible.